.github/workflows/release.yml

```diff
@@ -69,7 +83,7 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     needs: run_checker
-    if: needs.run_checker.outputs.run_release_charts == 'true'
+    if: github.ref_name == 'main' && needs.run_checker.outputs.run_release_charts == 'true'
```

Comment:
I realized while cleaning up the above that this would be deploying in merge queue, we should only ever run on main.